docs/public/nuxt-session.txt

```diff
@@ -0,0 +1 @@
+This is a test asset to move to the docs folder.
```

Comment:
should this go?

Reply:
I wanted to keep these in here to already have the public folder. I was scared that without it people would not know where to put their images for the docs.